test/gcloud/logging/project_test.rb

```diff
@@ -19,4 +19,15 @@
     logging.must_be_kind_of Gcloud::Logging::Project
     logging.project.must_equal project
   end
+
+  it "deletes a log" do
```

Comment:
It might be good to have a test that accepts a full log path instead of just the log name. Seeing as Entry will return the [full log path](https://cloud.google.com/logging/docs/api/ref_v2beta1/rest/v2beta1/LogEntry#FIELDS.log_name).